app/Commands/ListCommand.php

```diff
@@ -12,20 +12,24 @@ class ListCommand extends Command
 
     protected $signature = 'list {--json}';
     protected $description = 'List all services enabled by Takeout.';
+    protected $docker;
 
-    public function handle(): void
+    public function handle(Docker $docker): void
     {
+        $this->docker = $docker;
```

Comment:
Since it's all in the `handle` method, let's just keep it simple and make the call directly on the `$docker` object and skip assigning it as a class property.